### ISSUE TITLE
Download workspace with Eclipse support fails

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#1805]
I was all set to do some Maqetta to RAD testing. I created a new project with Eclipse support turned on and created a nymber of HTML files in Maqetta. Then, I went to download the entire workspace and get the following error when I click ok on the dialog (and the dialog never goes away and download never starts):

``` preformatted
Uncaught TypeError: Cannot call method 'getName' of undefined
Theme.isThemeHTML:50020/maqetta/app/davinci/davinci.js?34115103daf2016543a68f9effe58beaa18efc58:46965
_e34._rewriteUrls:50020/maqetta/app/davinci/davinci.js?34115103daf2016543a68f9effe58beaa18efc58:16831
_e34.okButton:50020/maqetta/app/davinci/davinci.js?34115103daf2016543a68f9effe58beaa18efc58:16847
_383dojo.js:4551
_492.(anonymous function)dojo.js:6092
declare._onClick:50020/maqetta/app/dijit/form/_ButtonMixin.js?34115103daf2016543a68f9effe58beaa18efc58:45
_25edojo.js:2636
declare._onClick:50020/maqetta/app/dijit/form/Button.js?34115103daf2016543a68f9effe58beaa18efc58:73
_383dojo.js:4551
_1e3:50020/maqetta/app/dijit/dijit.js?34115103daf2016543a68f9effe58beaa18efc58:1936
```

Environment: Chrome and lab9:50020

NOTE: As a follow-up test I went over to lab9:50010, and was able to download workspace with Eclipse support without issue (though I hadn't yet created any of my own files).

**Comment by aerwin** [comment]
Well, messing around some more, I got kicked out and had to login again. Now, it seems to be working on lab9:50020 as well. So, guess we can't reproduce, but obviously something was going wrong.

**Comment by JonFerraiolo** [comment]
@billreed63

**Comment by childsb** [comment]
this is working for me.  i created a few html files in an eclipse project, then downloaded. i also modified the library location before downloading and no error.

**Comment by jhpedemonte** [comment]
Reopening and moving to M6.

Ran into this issue while running the **Eclipse Project** test plan on **lab9:50020**, using Firefox 12/Mac.  Chose to **download the entire workspace** and took the defaults (all libs checked).  Debugger broke at this line:

```
return _2288.getName().indexOf("dojo-theme-editor.html")>-1;
```

in `isThemeHTML()`.  `_2288` is null.

We've now run into this bug on M4 (#1454), M5 and M6.

**Comment by childsb** [comment]
@billreed63 i believe this is failing after the clone theme fails.  download a fresh project with no cloned themes works fine.

**Comment by billreed63** [comment]
fixed by #2275
